config/crd/bases/placement.azure.com_clusterschedulingpolicysnapshots.yaml

```diff
@@ -144,6 +144,8 @@ spec:
                                     corresponding clusterSelectorTerm, in the range
                                     [-100, 100].
                                   format: int32
+                                  maximum: 100
```

Comment:
why is there changes in the snapshot CRD but I don't see change in its definition?

Reply:
i added the validation marker for the PreferredClusterSelector and it's part of PlacementPolicy in the spec of schedulingPolicySnapshot.